xml/deployment_sysreqs.xml

```diff
@@ -781,6 +781,250 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec.deploy.requirements.public_cloud">
+  <title>General Public Cloud Requirements</title>
+  <para>
```

Comment:
Add subsections for each cloud provider. Add the specific stuff per provider in the subsections.